binary_transparency/firmware/cmd/ft_witness/internal/http/witness_test.go

```diff
@@ -0,0 +1,61 @@
+package http
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/google/trillian-examples/binary_transparency/firmware/api"
+)
+
+func TestGetWitnessCheckpoint(t *testing.T) {
```

Comment:
Recommend adding a test for some failure cases. The DB not being initialized and failing to read / store would be useful tests.

Reply:
Yes, this is WIP..

Reply:
Done, added Failure use case for Witness and Storage Now. Please have a look!